prereise/gather/demanddata/bldg_electrification/floor_area_avg_slopes.py

```diff
@@ -0,0 +1,607 @@
+import os
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from zone_profile_generator import zone_shp_overlay
+
+from prereise.gather.demanddata.bldg_electrification import const
+
+
+def get_zone_floor_area(iso):
+    """Computes the zone floor area for each ISO.
+
+    :param str iso: abbrev. name of ISO.
+    :return: (*pandas.DataFrame*) zone_floor_area -- Floor area in square meters for all the zones with breakdowns of residential and commercial, total heating and total cooling
+    :note:: zone floor area in square meters saved as csv into Profiles/result_stats
```

Comment:
`:note::` -> `.. note::` We will also need an empty line before `note` section

Reply:
Made the changes